Toggl.Core.Tests/Interactors/TimeEntry/ContinueTimeEntryFromMainLogInteractorTests.cs

```diff
@@ -94,7 +94,7 @@ public async Task TracksTheTimeEntryContinuedEvent(ContinueTimeEntryMode continu
             InteractorFactory
                 .ContinueTimeEntry(Arg.Any<ITimeEntryPrototype>(), Arg.Any<ContinueTimeEntryMode>())
                 .Execute()
-                .Returns(Observable.Return(createdTimeEntry));
+                .Returns(Task.FromResult(createdTimeEntry));
```

Comment:
There's a helper method `ReturnsTaskOf`

```suggestion
                .ReturnsTaskOf(createdTimeEntry);
```